app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java

```diff
@@ -289,4 +293,57 @@ public void setTheme() {
         rb.setChecked(true);
         materialDialog.show();
     }
+
+    /**
+     * To set page number
+     */
+    private void setShowPageNumber() {
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        int currChoseId = mSharedPreferences.getInt (Constants.PREF_PAGE_STYLE_ID, -1);
+
+        RelativeLayout dialogLayout = (RelativeLayout) getLayoutInflater ()
+                .inflate (R.layout.add_pgnum_dialog, null);
+
+        RadioButton rbOpt1 = dialogLayout.findViewById(R.id.page_num_opt1);
+        RadioButton rbOpt2 = dialogLayout.findViewById(R.id.page_num_opt2);
+        RadioButton rbOpt3 = dialogLayout.findViewById(R.id.page_num_opt3);
+        RadioGroup rg = dialogLayout.findViewById(R.id.radioGroup);
+        CheckBox cbDefault = dialogLayout.findViewById (R.id.set_as_default);
+
+        if (currChoseId > 0) {
+            cbDefault.setChecked (true);
+            rg.clearCheck ();
+            rg.check (currChoseId);
+        }
+
+        MaterialDialog materialDialog = new MaterialDialog.Builder(mActivity)
+                .title(R.string.choose_page_number_style)
+                .customView(dialogLayout, false)
+                .positiveText(R.string.ok)
+                .negativeText(R.string.cancel)
+                .neutralText(R.string.remove_dialog)
+                .onPositive(((dialog, which) -> {
+
+                    int id = rg.getCheckedRadioButtonId ();
+                    String style = null;
+                    if (id == rbOpt1.getId ()) {
+                        style = Constants.PG_NUM_STYLE_PAGE_X_OF_N;
+                    } else if (id == rbOpt2.getId ()) {
+                        style = Constants.PG_NUM_STYLE_X_OF_N;
+                    } else if (id == rbOpt3.getId ()) {
+                        style = Constants.PG_NUM_STYLE_X;
+                    }
+                    if (cbDefault.isChecked ()) {
+                        editor.putString (Constants.PREF_PAGE_STYLE, style);
+                        editor.putInt (Constants.PREF_PAGE_STYLE_ID, id);
+                        editor.commit ();
+                    } else {
+                        editor.putString (Constants.PREF_PAGE_STYLE, null);
+                        editor.putInt (Constants.PREF_PAGE_STYLE_ID, -1);
+                        editor.commit ();
+                    }
```

Comment:
Why is there a space after `putString` and `putInt`?

Reply:
copied the code from image to pdf fragment. will fix it